subprojects/degrees-and-programs/source/static/app/js/degreesAndProgramsApp.js

```diff
@@ -110,7 +110,7 @@ var chapman = chapman || {};
 				var form = $(this),
 					target = $(event.target);
 
-				if (target.attr('id').includes('keyword')) {
+				if (target.attr('id').indexOf('keyword') !== -1) {
```

Comment:
I'm sorry @tatwell, I must have missed the notification for this comment.

`includes` was producing an error in IE11. To be safe, I converted all instances to `indexOf`.